src/utilities.ts

```diff
@@ -78,7 +78,7 @@ const utilities = {
 	 * @return {object}
 	 *   Result of the pathfinding operation.
 	 */
-	getPath(startPosition: RoomPosition, endPosition, allowDanger = false, addOptions: GetPathOptions = {}) {
+	getPath(startPosition: RoomPosition, endPosition: RoomPosition, allowDanger: boolean = false, addOptions: GetPathOptions = {}) {
```

Comment:
As you can see in the build results, endPosition can not only be a room Position, but also a goal object (an object containing `pos` and `range`), so that would need to be changed.
I've recently fixed the CI build so you should be able to use those reasonable well now. Feel free to ignore linter errors at the moment, until I manage to clean that up as well.

Reply:
I'm not sure. I've also had cases where the type checks weren't run completely when building with rollup. Maybe it does some kind of caching.